R/1-functions_processing.R

```diff
@@ -215,12 +211,11 @@ nmr_import_peaks <- function(path, method, pattern = "*.csv$", quiet = FALSE) {
   }
 
   # Combine, filter, and clean the sampleID column
-  peaks_rawdat %>%
-    bind_rows() %>%
-    filter(ppm >= 0 & ppm <= 10) %>%
-    filter(Intensity > 0) %>%
-    filter(!is.na(ppm)) %>%
-    # filter(!Flags == "Weak") %>%
-    mutate(sampleID = gsub(".csv", "", sampleID, fixed = TRUE)) %>%
-    weak_as_tibble()
+  pdat <- do.call("rbind", peaks_rawdat)
+  # KP_TODO: keep this? See #28
+  pdat <- subset(pdat, pdat$ppm >= 0 & pdat$ppm <= 10)
```

Comment:
@kaizadp If this stays in the code, need to document this behavior in the function documentation!

Reply:
We should delete this, and include this filtering in our vignettes instead. Agreed with #28

Reply:
Sounds good. I am also removing
```r
  pdat <- subset(pdat, pdat$Intensity > 0)
  pdat <- subset(pdat, !is.na(ppm))
```
unless you think otherwise.